packages/ci/src/lib/portal/transform.ts

```diff
@@ -71,6 +70,9 @@ function transformGQLCategory(category: CategoryFragment): CategoryConfig {
         weight,
       }),
     ),
+    // TODO: Portal API migration - convert isBinary to scoreTarget for backward compatibility
+    // Remove this conversion when Portal API supports scoreTarget (#713)
+    scoreTarget: category.isBinary ? 1 : undefined,
```

Comment:
It's a bit cleaner to omit the property altogether than to add it as `undefined`.

```suggestion
    ...category.isBinary && { scoreTarget: 1 },
```